samples/apps/agent-id-sample/frontend/src/styles.css

```
@@ -0,0 +1,2712 @@
+:root {
+  color: #111827;
+  background: #ffffff;
+  font-family:
+    Aptos, "Avenir Next", "SF Pro Text", "Helvetica Neue", ui-sans-serif,
+    system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", sans-serif;
+  font-synthesis: none;
+  text-rendering: optimizeLegibility;
```

Stylelint (17.11.0)

[error] Expected "optimizeLegibility" to be "optimizelegibility" (value-keyword-case)

(value-keyword-case)